src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/RoutingTests.cs

```diff
@@ -1,44 +1,136 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Routing;
 using Newtonsoft.Json;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Mvc.FunctionalTests
 {
-    public class RoutingTests : RoutingTestsBase<RoutingWebSite.StartupWith21Compat>
+    public class RoutingTests : IClassFixture<MvcTestFixture<RoutingWebSite.Startup>>
```

Comment:
What I did here is more like renaming `RoutingTestsBase` to `RoutingTests` and using the `RoutingEndpointRoutingTest` choice of `Startup` class. This one is messy because `RoutingEndpointRoutingTest` contained a fair number of additional tests as well as `override`s.